src/freenet/config/SubConfig.java

```diff
@@ -172,47 +172,53 @@ public int getInt(String optionName) {
 		synchronized(this) {
 			o = (IntOption) map.get(optionName);
 		}
-		return o.getValue();
+		// return fallback value for ignored options (null). This avoids breaking plugins which try to get ignored options.
+		return o == null ? -1 : o.getValue();
```

Comment:
Can we replace the ternaries with the more descriptive `Map.getOrDefault`?

Reply:
I tried: not without increasing the complexity, because map returns an option which does not have a simple constructor from just a value.

Reply:
Ah, yeah. I kind of forgot about the mess the constructor of Option is :disappointed: 

Spoiler:
```java
Option(SubConfig config, String name, ConfigCallback<T> cb, int sortOrder, boolean expert, boolean forceWrite,
	String shortDesc, String longDesc, DataType dataType) {
```

Well then let's stick with the ternaries!